src/sidebar/components/Annotation/AnnotationTimestamps.js

```diff
@@ -90,7 +90,7 @@ export default function AnnotationTimestamps({
       )}
       {annotationUrl ? (
         <Link
-          classes="p-link--muted p-link--hover-muted"
+          classes="text-color-text-light hover:text-color-text-light hover:underline"
```

Comment:
Yes, this styling does still represent a "pattern;" I'll come back and assess later.